draft-ietf-httpbis-http2bis.xml

```diff
@@ -3070,7 +3070,7 @@ cookie: e=f
         <reference anchor="RFC9112" target="https://www.rfc-editor.org/info/rfc9112">
           <front>
             <title>HTTP/1.1</title>
-            <author fullname="Roy T. Fielding" role="editor">
+            <author fullname="Roy Fielding" role="editor">
```

Comment:
The fullname accordng to RFC9112 is "Roy T. Fielding". The front matter disagrees. If you want to be ultra-correct, specify surname and initials separately.